docs/network-iot/run-an-lns/advanced-configuration.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: advanced-configuration
+title: Advanced Configuration
+pagination_label: Advanced Configuration
+sidebar_label: Advanced Configuration
+description: Advanced Configuration
+image: https://docs.helium.com/img/link-image.png
+slug: /iot/run-an-lns/advanced-configuration
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+## Session Key Filter (SKF)
+
+Session key filters enable you to allow or block device traffic on specific devAddrs using LoRaWAN
+device session keys.
+
```

Comment:
It would be good to mention why you'd need this; what problem does it solve. Something like:

If you don't configure SKF's for your devices you'll run the risk of someone trying to game the helium network and use your devaddrs to spoof traffic. If you don't have SKF's configured this works because in that case the helium network only uses the devaddr to identify the packet as belonging to your oui. And you'll pay for this packet.
A malicious gateway owner might use this trick to game the system and get more DC rewards.

SKF's can be used to mitigate this risk as with SKF's the helium packet router has enough information to verify if it is a valid packet and reject these invalid packets.